### ISSUE TITLE
node:https breaking for older node versions

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2876]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.46.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.46.0...4.46.1) (2022-09-06)

- 🚑 fix #2876 [`#2876`](https://github.com/open-wa/wa-automate-nodejs/issues/2876)
- Release 4.46.1 [`32c4ea5`](https://github.com/open-wa/wa-automate-nodejs/commit/32c4ea561cbb72fcef1a0036256100a1e858c71e)
- updated types-only package [`57ec718`](https://github.com/open-wa/wa-automate-nodejs/commit/57ec71808d67977ccfd16ef6250031da528fdc80)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.46.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.46.1...4.46.2) (2022-09-06)

- 🚑 fix #2876 [`#2876`](https://github.com/open-wa/wa-automate-nodejs/issues/2876)
- Release 4.46.2 [`e3dc82a`](https://github.com/open-wa/wa-automate-nodejs/commit/e3dc82a83155100fe286cc01e8edd47ed3c3d061)
- updated types-only package [`6b04be8`](https://github.com/open-wa/wa-automate-nodejs/commit/6b04be87e2dc61723db2c7ffa56948fd69af45ab)
